### PR TITLE
feat(EZPPFarm): Add Presence

### PR DESCRIPTION
**Pull request by ThomasCatlow**
**Describe the changes in this pull request:**

Add presence for EZPPFarm, an osu! private server.

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>

</details>

### CODE REVIEW

**Review by Slowlife01 — DISMISSED** [review]
Run prettier on your code

File: websites/E/EZPPFarm/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.5",
+  "author": {
+    "name": "Thomas",
+    "id": "300641413652021249"
+  },
+  "service": "EZPPFarm",
+  "altnames": ["EZPP"],
```
Comment (Bas950) [inline websites/E/EZPPFarm/dist/metadata.json:8]: Unneeded altname

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/E/EZPPFarm/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.5",
```
Comment (Bas950) [inline websites/E/EZPPFarm/dist/metadata.json:2]:
```suggestion
  "$schema": "https://schemas.premid.app/metadata/1.6",
```

**Review by Bas950 — DISMISSED** [review]

File: websites/E/EZPPFarm/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "url": [
+    "ez-pp.farm",
+    "new.ez-pp.farm",
+    "status.ez-pp.farm",
+    "git.ez-pp.farm",
+    "old.ez-pp.farm"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/PeNWQYB.png",
```
Comment (EncryptedDev) [inline websites/E/EZPPFarm/dist/metadata.json:19]: Not 512x512

File: websites/E/EZPPFarm/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  "description": {
+    "en": "EZPPFarm is an osu! Private Server featuring Vanilla, Relax, AutoPilot and ScoreV2 leaderboards, custom PP systems, and a massive community between our osu! and Discord server!"
+  },
+  "url": [
+    "ez-pp.farm",
+    "new.ez-pp.farm",
+    "status.ez-pp.farm",
+    "git.ez-pp.farm",
+    "old.ez-pp.farm"
+  ],
```
Comment (EncryptedDev) [inline websites/E/EZPPFarm/dist/metadata.json:17]: Use RegEx?

File: websites/E/EZPPFarm/presence.ts [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+  browsingStamp = Math.floor(Date.now() / 1000);
+let user: HTMLElement,
+  rank: HTMLElement,
+  pp: HTMLElement,
+  title: HTMLElement,
+  tag: HTMLElement,
+  url: URL,
+  mode: number,
+  id: number,
+  idURL: string;
```
Comment (EncryptedDev) [inline websites/E/EZPPFarm/presence.ts:11]: Do not let values which don't change, use constants where the value is assigned

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Comment by Slowlife01** [comment]
Inactive

**Pull request closed without merging**